lib/game.rb

```diff
@@ -1,12 +1,13 @@
 class Game
   CLI_GRID = <<~GRID.freeze
```

Comment:
I didn't check out the finished result for moving to < 5 line methods. I ended up extracting this into a template file instead of doing a constant.

One of the things that I learned from Sandi Metz, and also I think Avdi Grimm is converting constants to class methods. They are much more mockable, but just also free us up, like with attr_accessor to stop thinking about data and treat everything like a method.

Reply:
Yeah, putting it into a constant felt like technically not breaking the rule, but still felt wrong. I ended up moving it to a template in one of the PRs down the road.